vike/node/plugin/plugins/importUserCode/v1-design/getVikeConfig.ts

```diff
@@ -1111,20 +1116,48 @@ async function findPlusFiles(
     assertPosixPath(dir)
     ignorePatterns.push(`${path.posix.relative(userRootDir, dir)}/**`)
   }
-  const result = await glob(`**/+*.${scriptFileExtensions}`, {
-    ignore: [
-      '**/node_modules/**',
-      // Allow:
-      // ```
-      // +Page.js
-      // +Page.telefunc.js
-      // ```
-      '**/*.telefunc.*',
-      ...ignorePatterns
-    ],
-    cwd: userRootDir,
-    dot: false
-  })
+
+  let result: string[] = []
+
+  // If there is no .git folder, this will error
+  try {
+    const gitIncludePatterns = scriptFileExtensionList.map((ext) => `"**/+*.${ext}"`)
+    // -o lists untracked files only(but using .gitignore because --exclude-standard)
+    // -s adds the tracked files to the output
+    const { stdout } = await execA(
+      `git ls-files ${gitIncludePatterns.join(' ')} -os --exclude-standard --exclude="**/node_modules/**"`,
+      {
+        cwd: userRootDir
+      }
+    )
+
+    result = stdout
+      .split('\n')
+      .map((line) => line.split('\t').pop() || '')
```

Comment:
It seems like you need this because you are using `-os --exclude-standard`. It seems like everything is fine if you use `-oc` instead. Do you confirm?

Reply:
-oc works, nice